### ISSUE TITLE
Infinity or NaN toInt error due to custom pow() function when using Dart Sass and theme variables

### ISSUE THREAD

**Issue by sawyerh** [uswds/uswds#4220]
### Describe the bug

When setting a theme variable, like `$theme-color-primary: #ff0000;`, and when using Dart Sass to compile the file, the following error occurs: 

```
Error: Infinity or NaN toInt
     ╷
1044 │   $lum: round($lum * 1000) / 1000;
     │         ^^^^^^^^^^^^^^^^^^
     ╵
  node_modules/uswds/dist/scss/core/_functions.scss 1044:9              luminance()
  node_modules/uswds/dist/scss/core/_functions.scss 1356:11             calculate-grade()
  node_modules/uswds/dist/scss/core/_functions.scss 1908:14             get-color-token-from-bg()
  node_modules/uswds/dist/scss/core/mixins/_set-text-from-bg.scss 8:28  set-text-from-bg()
  node_modules/uswds/dist/scss/core/mixins/_set-text-from-bg.scss 25:3  set-text-and-bg()
  node_modules/uswds/dist/scss/elements/_buttons.scss 11:3              @import
  node_modules/uswds/dist/scss/packages/_uswds-components.scss 10:9     @import
  node_modules/uswds/dist/scss/uswds.scss 13:9                          @import
```

I don't totally understand it, but through logging using `@debug`, I found this variable would get logged as `NaN`:

https://github.com/uswds/uswds/blob/d89ece4e59234fb11ce8137041e58196ef32d45b/src/stylesheets/core/_functions.scss#L1031

### Steps to reproduce the bug

Repo with steps to reproduce this: https://github.com/sawyerh/uswds-dart-pow-bug

### Additional context

USWDS includes a range of custom math functions, including [its own definition of `pow()`](https://github.com/uswds/uswds/blob/d89ece4e59234fb11ce8137041e58196ef32d45b/src/stylesheets/core/_functions.scss#L939). 

Since Dart Sass 1.25.0, there's a [built-in `sass:math` module](https://sass-lang.com/documentation/modules/math) with `pow()` and other functions. **When importing the built-in `sass:math` module, and removing the custom USWDS `pow()` function, I do not get the `Infinity or NaN toInt` error and the Sass compiles to CSS as expected.**

### System setup

Dart Sass version: 1.34.1

- **USWDS version:** 2.11.2
- **Device:** MacBook Pro (13-inch, M1, 2020)
- **Operating system:** Big Sur
- **Browser and version:** N/A

**Comment by mejiaj** [comment]
Thanks for reporting this added it to the backlog. Related to #4204.

**Comment by mejiaj** [comment]
Hey @sawyerh, are you still having trouble with this? We've tried to reproduce without any luck. 

If you're not, we'd like to close this issue. Thanks!

**Comment by sawyerh** [comment]
@mejiaj Yes, still seeing this with [my reproduction here](https://github.com/sawyerh/uswds-dart-pow-bug), using the latest version of Dart Sass (`1.43.4`)

<img width="777" alt="CleanShot 2021-11-09 at 18 36 18@2x" src="https://user-images.githubusercontent.com/371943/141022786-e118b6ad-9612-4c92-9acb-32797b2b9241.png">

`sass test.scss` output:

<img width="887" alt="CleanShot 2021-11-09 at 18 37 09@2x" src="https://user-images.githubusercontent.com/371943/141022818-799ccaac-8be1-4cd8-8b69-c025a8eebd37.png">

**Comment by amyleadem** [comment]
Hello @sawyerh, this issue was fixed in the latest 2.13.1 release. 
More details about the change are available in the [release notes](https://github.com/uswds/uswds/releases/tag/v2.13.1):
`Prevent infinite loops in divide function. We improved the logic of the divide function to abort immediately on an attempt to divide by zero. Thanks [@aduth](https://github.com/aduth)! ([#4438](https://github.com/uswds/uswds/pull/4438))`

This issue should be safe to close.
